helpers/mathjax.js

```diff
@@ -94,21 +98,17 @@ export function loadMathJax(mathJaxConfig) {
 
 	if (mathJaxLoaded) return mathJaxLoaded;
 
-	const loadOptions = ['ui/menu'];
-	if (mathJaxConfig && mathJaxConfig.renderLatex) {
-		loadOptions.push('[tex]/all-packages');
-	}
-
 	window.MathJax = {
 		chtml: {
+			adaptiveCSS: false,
```

Comment:
`adaptiveCSS` is a neat feature MathJax introduced in 3.2 that allows its CSS stylesheets to be constructed on the fly as math is processed (so not _all_ of the styles need to be appended always). Unfortunately, we can't make use of it. MathJax keeps a single internal stylesheet reference for all referenced math, so if we turn this on, we're going to end up with inconsistent/incomplete styles in some components.

Reply:
It is unfortunate that it doesn't just append more styles to it, which could then be used in the various scopes.

Reply:
Yeah, MathJax seems to rely heavily on globally scoping everything and sort of just building out whatever it needs when it needs it. Which is fine most of the time, but completely breaks in a situation like ours where we effectively have multiple documents with different math needs. 😞